app/routers/chem.py

```diff
@@ -68,7 +68,9 @@ async def SMILES_to_Stereo_Isomers(smiles: str):
 
 @router.get("/descriptors")
 async def SMILES_Descriptors(
-    smiles: str, format: Optional[str] = "json", toolkit: Optional[str] = "rdkit"
+    smiles: str,
+    format: Literal["json", "html"],
+    toolkit: Literal["cdk", "rdkit", "all"],
```

Comment:
```suggestion
    toolkit: Literal["cdk", "rdkit", "all"] = "rdkit",
```

you still want to add in the defaults here, or even better with proper FastAPI annotations:

```suggestion
    toolkit: Literal["cdk", "rdkit", "all"] = fastapi.Query(default="rdkit", description="The cheminformatics toolkit to use as a backend")
```

disclaimer: code in browser, semantics might be a bit different

Reply:
Should be fixed with #302